src/app/features/sidebar/pools-section/pool-item.tsx

```diff
@@ -24,18 +25,22 @@ const PoolItem = ({styles, data, state, handlers}) => {
         handlers.edit()
       },
     },
+    {
+      label: "Get Info",
+      click: () => {
+        dispatch(Tabs.activateUrl(lakePoolPath(pool.id, lakeId)))
+      },
+    },
```

Comment:
I also made the clicking of a pool create a draft query with that pool as a pin. Otherwise, you can right click and click "get info" to pull up the pool page.